Slim/Player/StreamingController.pm

```diff
@@ -948,8 +960,6 @@ sub _Continue {
 			$song->setStatus(Slim::Player::Song::STATUS_PLAYING);
 		}
 	} else {
```

Comment:
Is there a reason for removing that comment?

Reply:
No. Reverting.